qa-tests-backend/src/test/groovy/IntegrationsTest.groovy

```diff
@@ -527,6 +527,31 @@ class IntegrationsTest extends BaseSpecification {
                 Env.mustGetGCPAccessKey()
     }
 
+    @Unroll
+    @Tag("Integration")
+    def "Verify GCS Integration: #integrationName"() {
+        setup:
+        Assume.assumeTrue(!useWorkloadId || Env.HAS_WORKLOAD_IDENTITIES)
+
+        when:
+        "the integration is tested"
+        def backup = ExternalBackupService.getGCSIntegrationConfig(integrationName, useWorkloadId)
+
+        then:
+        "verify test integration"
+        // Test integration for GCS performs test backup (and rollback).
+        assert ExternalBackupService.testExternalBackup(backup)
```

Comment:
nit: assert is not required in then block. What's more I'm not a fan of this method as it swallow exception and only log. How it looks when test fails? Is it part of the junit output or we need to look at logs? Do we need a retry on this?
```suggestion
        ExternalBackupService.testExternalBackup(backup)
```

Reply:
I don't think we need retries, the underlying storage client should already retry.